src/outfit.ts

```diff
@@ -340,6 +342,108 @@ function cheeses(embezzlerUp: boolean) {
       )
     : [];
 }
+
+function mafiaThumbRing(equipMode: BonusEquipMode) {
+  if (!have($item`mafia thumb ring`) || ["free", "dmt"].some((mode) => mode === equipMode)) {
+    return new Map<Item, number>([]);
+  }
+
+  return new Map<Item, number>([
+    [$item`mafia thumb ring`, (1 / 0.96 - 1) * get("valueOfAdventure")],
+  ]);
+}
+
+function luckyGoldRing(equipMode: BonusEquipMode) {
+  // Ignore for DMT, assuming mafia might get confused about the volcoino drop by the weird combats
+  if (!have($item`lucky gold ring`) || equipMode === "dmt") {
+    return new Map<Item, number>([]);
+  }
+
+  // Volcoino has a low drop rate which isn't accounted for here
+  // Overestimating until it drops is probably fine, don't @ me
+  const dropValues = [
+    100, // 80 - 120 meat
+    ...[
+      itemAmount($item`hobo nickel`) > 0 ? 100 : 0, // This should be closeted
+      itemAmount($item`sand dollar`) > 0 ? getSaleValue($item`sand dollar`) : 0, // This should be closeted
+      itemAmount($item`Freddy Kruegerand`) > 0
+        ? Math.max(
+            getSaleValue($item`bottle of Bloodweiser`) / 200,
+            getSaleValue($item`electric Kool-Aid`) / 200
```

Comment:
keys here too, i think?

Reply:
also, sometimes bloodweiser/kool-aid mallPrice gets a little off but not sure if we can do anything about that